src/plugins/walletConnectToDapps/components/DappRegistryGrid.tsx

```diff
@@ -0,0 +1,96 @@
+import { SearchIcon } from '@chakra-ui/icons'
+import {
+  Box,
+  Heading,
+  Image,
+  Input,
+  InputGroup,
+  InputLeftElement,
+  Link,
+  SimpleGrid,
+  Stack,
+  Text as PlainText
+} from '@chakra-ui/react'
+import { Text } from 'components/Text'
+import { FC, useEffect, useMemo } from 'react'
+import { useForm } from 'react-hook-form'
+
+interface RegistryItem {
+  category: string
+  id: string
+  homepage: string
+  name: string
+  image: string
+}
+
+const registry: RegistryItem[] = require('../registry.json')
```

Comment:
We can import JSON files with the ESM `import` syntax as well

Reply:
To honor the type name and make it easier to name items iterated over

```suggestion
const registryItems: RegistryItem[] = require('../registry.json')
```

Reply:
@gomesalexandre how would you get the variable typed properly when using import?